### ISSUE TITLE
macOS application not a Universal Binary

### ISSUE THREAD

**Issue by DaveFlashNL** [Jo2003/cd2netmd_gui#2]
![Schermafbeelding 2022-02-01 om 19 21 19](https://user-images.githubusercontent.com/72280178/152027635-582435e7-7f59-467a-857c-a4a92102314e.png)

please consider building next version as U2B, Universal Binary 2 for full native mode on M1-Apple Silicon Macs as I have.

**Comment by Jo2003** [comment]
Good idea, but since I don't have a M1 Mac I can't do this. Maybe you are able?

**Comment by DaveFlashNL** [comment]
intel Macs are able to compile for apple silicon: https://developer.apple.com/documentation/apple-silicon/building-a-universal-macos-binary

**Comment by DaveFlashNL** [comment]
i am wiling to test a build for you.

**Comment by Jo2003** [comment]
Maybe later when I find the time.

**Comment by Jo2003** [comment]
#response_container_BBPPID{font-family: initial; font-size:initial; color: initial;}  If you find the time and like to, please do it ;-)                                                                                                                                              Gesendet über BlackBerry Hub+ Posteingang für Android                                                                                                                                                                           Von: ***@***.***: 13. Februar 2022 21:11An: ***@***.***: ***@***.***: ***@***.***; ***@***.***: Re: [Jo2003/cd2netmd_gui] macOS application not a Universal Binary (Issue #2)  
Th release process could use the github actions, which is able to build for MacOS, I guess universal binaries would just be possible as well.
I will take a look into that.
